pkg/gatherers/clusterconfig/certificate_signing_requests.go

```diff
@@ -53,7 +59,9 @@ func (g *Gatherer) GatherCertificateSigningRequests(ctx context.Context) ([]reco
 func gatherCertificateSigningRequests(ctx context.Context,
 	certClient certificatesv1.CertificateSigningRequestsGetter) ([]record.Record, []error) {
 	requests, err := certClient.CertificateSigningRequests().List(ctx, metav1.ListOptions{
-		Limit: csrGatherLimit,
+		// limit the maximum number of crs that
+		// will be listed in a single request to reduce memory usage.
+		Limit: 5000,
```

Comment:
Another one that used to be a constant declared at the package global scoped and was only being used here.